index.html

```diff
@@ -1540,6 +1553,46 @@ <h2>Event summary</h2>
       </tbody>
     </table>
   </section>
+  <section class="informative">
+    <h2>Timestamp behavior</h2>
+    <h2>RTCRtpSender timestamp effects on packet NTP and RTP timestamps</h2>
```

Comment:
```suggestion
    <h2>RTP timestamp effects on packet NTP and RTP timestamps</h2>
```
there is nothing in the spec that says that incoming RTP is coming from an RTPSender.

Reply:
I've clarified the text, PTAL.